### PR TITLE
Bento 2283

### PR DESCRIPTION
**Pull request by rana22**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/pages/dashTemplate/tabs/TabView.js
Comment (iksheth) [inline src/pages/dashTemplate/tabs/TabView.js:1]: @rana22 Can we call this one as TabPanel?

File: src/bento-core/PaginationTable/AddToCartDialog/SuccessOutlined.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { createSvgIcon } from 'bento-components';
```
[the remark this answers is not in the document]
Comment (rana22) [reply src/bento-core/PaginationTable/AddToCartDialog/SuccessOutlined.js]: moved.

File: src/bento-core/Tab/TabViewGenerator.js
[the remark this answers is not in the document]
Comment (rana22) [reply src/bento-core/Tab/TabViewGenerator.js:1]: updated

File: src/pages/dashTemplate/tabs/TabsView.js
[the remark this answers is not in the document]
Comment (rana22) [reply src/pages/dashTemplate/tabs/TabsView.js:1]: updated

File: src/pages/dashTemplate/tabs/TabView.js
Comment (rana22) [reply src/pages/dashTemplate/tabs/TabView.js:1]: updated

**Review by iksheth — APPROVED** [review]
Looks Good to Me👍🏻

**Pull request merged**